adding Node.js + Express.js webhooks app example (#293)
miroapp/app-examples: examples/node-webhooks/README.md

  - [miro.as()](https://miroapp.github.io/api-clients/node/classes/index.Miro.html#as)
  - [api.getAllBoards()](https://miroapp.github.io/api-clients/node/classes/index.MiroApi.html#getAllBoards)

# 🛠️ Tools and Technologies <a name="tools"></a>

- [Node.js](https://nodejs.org/en)
- [Express.js](https://expressjs.com/)

# ✅ Prerequisites <a name="prerequisites"></a>

- You have a [Miro account](https://miro.com/signup/).
- You're [signed in to Miro](https://miro.com/login/).
- Your Miro account has a [Developer team](https://developers.miro.com/docs/create-a-developer-team).
- Your development environment includes [Node.js 14.13](https://nodejs.org/en/download) or a later version, and npm.
- Your development environment includes [ngrok](https://ngrok.com/) or something similar.

# 📖 Associated Developer Tutorial <a name="tutorial"></a>

> To view a more in depth developer tutorial
> of this app (including code explanations) see the [Getting started with webhooks tutorial](https://developers.miro.com/docs/getting-started-with-webhooks) on Miro's Developer documentation.

# 🏃🏽‍♂️ Run the app locally <a name="run"></a>

> 1. Create a new Miro app on [developers.miro.com](https://developers.miro.com/). This will take you to the app settings page. There you
>    will find the `MIRO_CLIENT_ID` and `MIRO_CLIENT_SECRET` to be added to your `.env` file. Ensure that `boards:read` scope is checked,
>    and then go ahead and install the app on your developer team. You will get an **access token** which you will need later to
>    authenticate the creation of your webhook subscription.

Reviewer: ```suggestion
1. **Create a Miro app** on [developers.miro.com](https://developers.miro.com/). This will take you to the app settings page, where you will find the `MIRO_CLIENT_ID` and `MIRO_CLIENT_SECRET`. These need to be added to your `.env` file.
   
   - Ensure the `boards:read` scope is checked.
   - Install the app on your developer team. You will get an **access token**, which is required later to authenticate your webhook subscription.
```

> 2. In a new terminal session, run:
> 
> ```
> ngrok http 3000
> ```
> 
> This will output something like this:
> 
> ```
> Forwarding https:<your-ngrok-url> -> http://localhost:3000
> ```
> 
> The `https:<your-ngrok-url>` is your `MIRO_REDIRECT_URL` to be used in the `.env` file and then later when calling the API to create a webhook subscription.
> 
> 3. Rename the `.sample.env` file to `.env` and then add in your `MIRO_CLIENT_ID` and `MIRO_CLIENT_SECRET` from your [developers.miro.com](https://developers.miro.com/) app settings page. Use the `forwarding URL` from the previous step for the `MIRO_REDIRECT_URL` in the .env file. Save the file as `.env` with your new variables.
> 
> 4. Run `npm i` to install dependencies.
> 
> 5. Run `npm start` to start the dev server.
> 
> 6. Go to your developer team, and open the board you want to receive webhook events for.
> 
> 7. In a separate browser tab, open up the API Exporer for the [Create Webhook Subscription endpoint](https://developers.miro.com/reference/create-board-subscription).
> 
> 8. Provide the following information in the API Explorer:
> 
> > **Access Token**: Once you get the access token after installing your app on a developer team (from step 4 above), you can add the access token to the Authorization section of the API reference page.
> >
> > **boardId:** Get the board ID of the board you want to receive notifications for. This board should be in the same team where you installed the app. You can find board ID in the URL when you go to your board: https://miro.com/app/board/<boardId>.
> >
> > **callbackUrl:** This is the URL where you will receive events. It should be the same as `MIRO_REDIRECT_URL` in `.env`.
> 
> 9. Select Try It! to run the API request right from the browser. If you get a 201 response, you are ready to receive events!
> 
> 10. Next, go to to the same board which you referenced in the request above, and create a sticky. You should now receive a webhook event! To learn more about the events you can expect to receive,

Reviewer: ```suggestion
2. In a new terminal session, run:

   ```
   ngrok http 3000
   ```

   This will output something like this:

   ```
   Forwarding https:<your-ngrok-url> -> http://localhost:3000
   ```

   The `https:<your-ngrok-url>` is your `MIRO_REDIRECT_URL` to be used in the `.env` file and then later when calling the API to create a webhook subscription.

3. Rename the `.sample.env` file to `.env` and then add in your `MIRO_CLIENT_ID` and `MIRO_CLIENT_SECRET` from your [developers.miro.com](https://developers.miro.com/) app settings page. Use the `forwarding URL` from the previous step for the `MIRO_REDIRECT_URL` in the `.env` file. Save the file as `.env` with your new variables.

4. Run `npm i` to install dependencies.

5. Run `npm start` to start the dev server.

6. Go to your developer team, and open the board you want to receive webhook events for.

7. In a separate browser tab, open the [Create Webhook Subscription](https://developers.miro.com/reference/create-board-subscription) endpoint.

8. Provide the following information:

   - **Access Token**: Once you get the access token after installing your app on a developer team in step 1, you can add the access token to the Authorization section of the API reference page.
   - **boardId:** Get the board ID of the board you want to receive notifications for. This board should be in the same team where you installed the app. You can find board ID in the URL when you go to your board: https://miro.com/app/board/<boardId>.
   - **callbackUrl:** This is the URL where you will receive events. It should be the same as `MIRO_REDIRECT_URL` in `.env`.

9. Select **Try It!** to run the API request right from the browser. If you get a `201` response, you are ready to receive events!

10. Next, go to to the same board which you referenced in the request above, and create a sticky. You should now receive a webhook event!
```
